pkg/util/oom/oom_linux.go

```diff
@@ -75,6 +75,7 @@ func applyOOMScoreAdj(pid int, oomScoreAdj int) error {
 			err = fmt.Errorf("failed to apply oom-score-adj to pid %d (%v)", pid, err)
 			continue
 		}
+		defer f.Close()
```

Comment:
Thanks for your comment, @smarterclayton . Surely we should return an error if the file wasn't written. But I'm not sure if the defer will prevent the error to be returned?